metadata-ingestion/src/datahub/ingestion/source/redshift/config.py

```diff
@@ -112,10 +113,6 @@ class RedshiftConfig(
         description="Whether target Redshift instance is serverless (alternative is provisioned cluster)",
     )
 
-    use_lineage_v2: bool = Field(
-        default=True,
-        description="Whether to use the new SQL-based lineage collector.",
-    )
     lineage_v2_generate_queries: bool = Field(
```

Comment:
It would be great to remove all user-facing references to v1/v2 so, what about renaming `lineage_v2_generate_queries` as `lineage_generate_queries`?

We are already introducing a breaking change, so maybe this is best moment for such a rename.